eventlog/models.py

```diff
@@ -0,0 +1,39 @@
+from django.db import models
+from uuid import uuid4
+from roster.models import Period, ClusiveUser, Roles
+from django.utils import timezone
+import caliper
+
+# A user session
+class Session(models.Model):
```

Comment:
I think we should name this something like `ClusiveSession`, `SessionEvent` or similar to avoid possible confusion / collision with the `Session` object in `django.contrib.sessions.models`.

Reply:
LoginSession is what we've used in previous projects, will change to that.